### PR TITLE
Feature/13246 appgw holding page

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Task: [AB#13246](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13246)
Activity: [AB#13466](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13466)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,6 @@
 
 /temp_modules
 
-*/\.vs/
\ No newline at end of file
+*/\.vs/
+
+/monitoring/scripts/ingress/*
```
Comment (bjss-chris) [inline .gitignore]: Trailing new line. 😃

File: monitoring/scripts/central-monitor-setup.ps1 [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+helm repo add rancher-stable https://releases.rancher.com/server-charts/stable
+helm repo update
+kubectl create namespace cattle-system
+helm install rancher rancher-stable/rancher --namespace cattle-system --set hostname=rancher.dynamic.buyingcatalogue.digital.nhs.uk --set tls=external
+
+start-sleep 120 # to allow rancher pods to deploy
+
+#ipArr=()
+#        ipArr+=$(kubectl get pods -l app=rancher -n cattle-system -o wide | while read line ; do while read a b c d e ip g h i; do echo $ip ; done ; done)
+#        az network application-gateway address-pool update -g gpitfutures-monitor-rg-appgw --gateway-name gpitfutures-monitor-appgw -n rancher --servers $ipArr
```
Comment (bjss-chris) [inline monitoring/scripts/central-monitor-setup.ps1]: Trailing new line. 😃

**Review by bjss-chris — CHANGES_REQUESTED** [review]

File: monitoring/scripts/central-monitor-setup.ps1 [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+helm repo add kubernetes-dashboard https://kubernetes.github.io/dashboard/
+helm install kubernetes-dashboard kubernetes-dashboard/kubernetes-dashboard --create-namespace --namespace kubernetes-dashboard
+
+kubectl create namespace ingress-appgw
+
+cd <path-to-repo>\platform-tf\monitoring\
+
+kubectl create -f "../ingress/deployment-rbac.yaml"
+
+#az identity list -g gpitfutures-monitor-rg-aks -o json > ./ingress/monitor-msi.txt
```
Comment (PhysicsJon) [inline monitoring/scripts/central-monitor-setup.ps1]: Is this being reenabled in the future or can it be removed?

File: monitoring/scripts/central-monitor-setup.ps1 [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+kubectl apply -f ./scripts/ingress/rancher-monitor-azureingressprohibitedtarget.yaml -n ingress-appgw
+
+helm repo add rancher-stable https://releases.rancher.com/server-charts/stable
+helm repo update
+kubectl create namespace cattle-system
+helm install rancher rancher-stable/rancher --namespace cattle-system --set hostname=rancher.dynamic.buyingcatalogue.digital.nhs.uk --set tls=external
+
+start-sleep 120 # to allow rancher pods to deploy
+
+#ipArr=()
```
Comment (PhysicsJon) [inline monitoring/scripts/central-monitor-setup.ps1]: Is this being reenabled in the future or can it be removed?

**Review by PhysicsJon — CHANGES_REQUESTED** [review]

File: monitoring/scripts/central-monitor-setup.ps1 (hunk 3 above)
Comment (gavinwakeley1) [reply monitoring/scripts/central-monitor-setup.ps1]: Don't really see why we do it at all - I've removed the line. Thanks

File: .gitignore (hunk 1 above)
Comment (gavinwakeley1) [reply .gitignore]: Added, thanks!

File: monitoring/scripts/central-monitor-setup.ps1 (hunk 2 above)
Comment (gavinwakeley1) [reply monitoring/scripts/central-monitor-setup.ps1]: Replaced by PS1 version and #'d back in - thanks!

**Review by bjss-chris — APPROVED** [review]

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**